src/pr-gating/Setup.md

```diff
@@ -26,17 +26,19 @@
 
   If you have more branches that should be protected, you can setup the same check for those branches.
-  
+
   <img width="800" alt="buildpolicy" src="https://github.com/microsoft/GHAzDO-Resources/assets/106392052/74801e80-46e1-4d05-97b1-5f11396330e1">
 
 
-- Figure out a way to test your new setup without adding any bad code to your main branch :) 
+- Figure out a way to test your new setup without adding any bad code to your main branch :)
 
 ## Possible customizations
-- The CodeQL language (javascript) is hardcoded in CIVerify.yml. GHAzDO settings will need to be updated to match your needs. 
-- The CI Verification could be extended to also (or only) check new Dependency alerts. The same strategy could be used but tweaks would have to be done to both CIVerify.yml and CIGate.ps1. 
-One complication to concider if you want to use the same strategy for dependency scanning. It could be that the reason we see new issues reported is that the advisory database has been extended since the last scan of main. That is, the new additions in the PR is not the issue but rather that this scan is the first one after an extention of the known vunarabilities. Still, since the alerts can be dismissed in the PR branch, the branch policy check can be setup as optional, or a new scan can be run on the main branch, I think this is acceptable. 
+- The CodeQL language (javascript) is hardcoded in CIVerify.yml. GHAzDO tasks will need to be updated to match your needs.
+- CIGate.ps1 contains a policy coded for which alerts types (`code` and/or `dependency`) and vulnerability/quality severeties ( `critical`, `high`, `medium`, `low`, `error`, `warning`, and `note`).  Customize this in the script as needed to meet your policy thresholds (example: do not fail on Quality checks or Low severity security issues by removing `low`, `error`, `warning`, and `note` )
+- One complication to concider for new alerts appearing for Code or Dependency scanning. It could be that the reason we see new issues reported is that this PR is the first time scanning using updated CodeQL tooling or the GitHub Advisory Database has been extended since the last scan of main. That is, the new additions in the PR is not the issue but rather that this scan is the first one after an extention of the known vunarabilities. Still, since the alerts can be dismissed in the PR branch, the branch policy check can be setup as optional, or a new scan can be run on the main branch, I think this is acceptable.
+- dynamically pass/fail the build only if a PR comment was added, indicating that there are new alerts that were directly created by this PR.  Since we do incremental PR iteration based comments this is not currently viable without a rewrite
```

Comment:
```suggestion
- Dynamically pass/fail the build only if a PR comment was added, indicating that there are new alerts that were directly created by this PR.  Since we do incremental PR iteration based comments this is not currently viable without a rewrite
```